### ISSUE TITLE
Accessibility : Focus is not going to "searchForMoreText" prop value in the Tag Picker Suggestion pop up in windows high contrast black and white mode

### ISSUE THREAD

**Issue by Sandhya-aluri** [microsoft/fluentui#13879]
### Environment Information

- **Package version(s)**: "office-ui-fabric-react": "^7.94.1"
- **Browser and OS versions**: chromium Edge , Windows 10

### Describe the issue:
We are using office fabric Tag Picker control. After applying high contrast black and white mode, the keyboard focus is not visible on the “Show more”(search For more text/Load all results) button in the Tag Picker while navigating using down arrow key.

#### Actual behavior:

Keyboard focus should be clearly visible on the “show more/Load all results” button in the Tag picker control after applying high contrast black and white mode.

#### Expected behavior:

After applying high contrast black and white mode, the keyboard focus is not visible on the “Show more/Load all results
<img width="316" alt="Capture" src="https://user-images.githubusercontent.com/48669559/86227020-1fc41500-bbaa-11ea-9b20-121259c1f2ec.PNG">
”(searchFormoretext) button in the Tag Picker while navigating using down arrow key.

### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low)

Products/sites affected: (if applicable)

**Comment by paulgildea** [comment]
Thanks for the repro @Sandhya-aluri 

@xugao - Not sure if this is a pure High Contrast issue - so adding component owner

**Comment by xugao** [comment]
@Sandhya-aluri - can you share your codepen link for repro?

**Comment by Sandhya-aluri** [comment]
@xugao : yes it is the high contrast issue in black and white mode. Please find the below link of code pen and check in edge browser high contrast black mode and white mode.
https://codepen.io/sagsah/pen/RwWxxpw?editors=1000

**Comment by xugao** [comment]
@Sandhya-aluri - thanks for the repro! will look into fixing
